packages/cbor/src/main/cbor/Decoder.ts

```diff
@@ -35,12 +35,14 @@ export interface DecoderConfig {
   max_text_length: number;
 }
 
+const MEGABYTE_IN_BYTES = 0x100000;
```

Comment:
Sounds a bit confusing TBH.. Isn't the other way around makes more sense (or just `MEGABYTE`)?

(SECONDS_IN_MINUTE, DAYS_IN_WEEK, etc..)

Reply:
Yeah, you're right. First I wanted to write `ONE_MEGABYTE_IN_BYTES` but it makes more sense the other way round.